lib/zendesk_apps_tools/server.rb

```diff
@@ -17,6 +17,7 @@ class Server < Sinatra::Base
           last_mtime = curr_mtime
         end
       end
+      settings.parameters.merge!(locale: params['locale'])
```

Comment:
An edge case would be if the app also names a parameter 'locale', then we're going to overwrite it here. Can we pass it through without touching parameters?